### PR TITLE
Add Quefrency Rev. 6

### PR DESCRIPTION
**Pull request by nooges**
## Description

Add Quefrency Rev. 6

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keebio/quefrency/rev6/config.h [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+#define SERIAL_USART_RX_PAL_MODE 7
+#define SERIAL_USART_FULL_DUPLEX
+#define SERIAL_USART_PIN_SWAP
+
+#define USB_VBUS_PIN C6
+#define SPLIT_HAND_PIN A0
+
+/* Defines for the RGB matrix */
+#ifdef RGB_MATRIX_ENABLE
+#   define NOP_FUDGE 0.4
```
Comment (waffle87) [inline keyboards/keebio/quefrency/rev6/config.h:20]: This shouldn't be needed? Compiles fine without and it appears you have WS2812 driver and the like configured appropriately.

File: keyboards/keebio/quefrency/rev6/rules.mk [hunk 2]
```diff
@@ -0,0 +1,3 @@
+SERIAL_DRIVER = usart
+EEPROM_DRIVER = wear_leveling
+WEAR_LEVELING_DRIVER = embedded_flash
```
Comment (waffle87) [inline keyboards/keebio/quefrency/rev6/rules.mk:3]: This is the default behaviour.
```suggestion
```

File: keyboards/keebio/quefrency/rev6/config.h (hunk 1 above)
Comment (nooges) [reply keyboards/keebio/quefrency/rev6/config.h]: Removed it now after testing w/o it

File: keyboards/keebio/quefrency/rev6/rules.mk (hunk 2 above)
Comment (nooges) [reply keyboards/keebio/quefrency/rev6/rules.mk:3]: Hmm, it failed to compile either of those lines

Comment (waffle87) [reply keyboards/keebio/quefrency/rev6/rules.mk:3]: Odd, pretty sure I'm interpreting the default correctly? Someone else feel free to chime in.
https://github.com/qmk/qmk_firmware/blob/fe50774cb488f981dfd41c7a5bc5f34dfcd9cb9d/builddefs/common_features.mk#L218-L222

Comment (zvecr) [reply keyboards/keebio/quefrency/rev6/rules.mk:3]: For me, compiles fine without those lines.

If it works, :shrug:

Comment (nooges) [reply keyboards/keebio/quefrency/rev6/rules.mk]: Removed those lines now that I've rebased onto the latest master with the fix

**Review by waffle87 — APPROVED** [review]

File: keyboards/keebio/quefrency/rev6/keyboard.json [hunk 3]
```diff
@@ -0,0 +1,633 @@
...
+            {"matrix": [9, 8], "x": 224, "y": 64, "flags": 4},
+            {"x": 224, "y": 52, "flags": 2}
+        ],
+        "max_brightness": 120,
+        "sleep": true,
+        "split_count": [50, 51]
+    },
+    "split": {
+        "bootmagic": {
+            "matrix": [0, 0]
```
Comment (zvecr) [inline keyboards/keebio/quefrency/rev6/keyboard.json:193]: The intention with this define is to map it to something on the right hand, so i would expect it to be something like `[5, 8]`?

Comment (nooges) [reply keyboards/keebio/quefrency/rev6/keyboard.json:193]: Ah yes, will revise, test it, and commit change

**Review by zvecr — APPROVED** [review]

**Pull request merged**